src/ralph/settings/base.py

```diff
@@ -144,6 +144,8 @@ def str_to_bool(s: str) -> bool:
         'ATOMIC_REQUESTS': True,
         'TEST': {
             'NAME': 'test_ralph_ng',
+            'CHARSET': 'utf8',
+            'COLLATION': 'utf8_general_ci',
```

Comment:
unfortunately, `AssertionError: PostgreSQL does not support collation setting at database creation time`